### ISSUE TITLE
Error Code 124 > invalid access token

### ISSUE THREAD

**Issue by hdytarsn** [MacsiDigital/laravel-zoom#22]
I'm trying to get list of users for test api.
I'm getting 401 status code.
After im add  `dd($this->response->getBody());` to **find** function in **model.php**
after i'm gettin following response from zoom.
array:2 [▼
  "code" => 124
  "message" => "Invalid access token."
]

I'm using laravel 7.0 on digital ocean ubuntu droplet.
what could be the problem? 
Thanks

**Comment by ivanmv1987** [comment]
I solve this putting in the config file these
'zoom.api_key' => env('ZOOM_KEY'),
'zoom.api_secret' => env('ZOOM_SECRET'),

> I'm trying to get list of users for test api.
> I'm getting 401 status code.
> After im add `dd($this->response->getBody());` to **find** function in **model.php**
> after i'm gettin following response from zoom.
> array:2 [▼
> "code" => 124
> "message" => "Invalid access token."
> ]
> 
> I'm using laravel 7.0 on digital ocean ubuntu droplet.
> what could be the problem?
> Thanks

**Comment by hdytarsn** [comment]
Which config file did you put?

**Comment by ivanmv1987** [comment]
inside the directory
config/zoom.php

> Which config file did you put?

**Comment by colinhall17** [comment]
Hi

Apologies in the delay in coming back to you, when I created this package it was a quick build for a project I needed it for and I made it open source in case it would help someone. 

With the increased use of Zoom around the world its gaining some traction, so I've built 3 new versions for Laravel 5.5 upwards, make sure you select the correct version for your version of Laravel.

Make sure you follow the documentation to download the zoom config file.  Documentation is now more helpful also on how to achieve most things.

Open a new issue with any problems.
